### ISSUE TITLE
Unable to update using `aio update` to update to 3.4.1 from 3.3.0

### ISSUE THREAD

**Issue by sarahxxu** [adobe/aio-cli#164]
### Expected Behaviour

`aio update` should install the latest version of the CLI. 

### Actual Behaviour

`aio update` does not seem to update my Adobe I/O CLI properly, even though the result seems successful. I had to use `npm` to update. 

```
sarah:~ sarahxxu$ aio update
Plugin(s) to update Current   Latest 
@adobe/aio-cli      3.3.0     3.4.1  

? Update 1 plugin(s)? Yes
warning "@adobe/aio-cli-plugin-app > request-promise@4.2.5" has unmet peer dependency "request@^2.34".
warning "@adobe/aio-cli-plugin-app > @adobe/aio-lib-ims > @adobe/aio-lib-ims-jwt@1.0.0" has incorrect peer dependency "@adobe/aio-lib-ims@^1.0.0".
warning "@adobe/aio-cli-plugin-app > @adobe/generator-aio-console > inquirer-autocomplete-prompt@1.0.2" has incorrect peer dependency "inquirer@^5.0.0 || ^6.0.0".
warning "@adobe/aio-cli-plugin-app > request-promise > request-promise-core@1.1.3" has unmet peer dependency "request@^2.34".
Installing plugin @adobe/aio-cli... installed v3.4.1

sarah:~ sarahxxu$ aio -v 
@adobe/aio-cli/3.3.0 darwin-x64 node-v12.16.3

sarah:~ sarahxxu$ npm install -g @adobe/aio-cli
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated @adobe/aio-cna-core-config@1.0.14: WARNING: This module has been renamed to @adobe/aio-lib-core-config. Please install it instead. See https://github.com/adobe/aio-lib-core-config for more information.
/usr/local/bin/aio -> /usr/local/lib/node_modules/@adobe/aio-cli/bin/run
/usr/local/bin/adobe -> /usr/local/lib/node_modules/@adobe/aio-cli/bin/run

> fsevents@1.2.13 install /usr/local/lib/node_modules/@adobe/aio-cli/node_modules/watchpack-chokidar2/node_modules/fsevents
> node install.js

  SOLINK_MODULE(target) Release/.node
  CXX(target) Release/obj.target/fse/fsevents.o
  SOLINK_MODULE(target) Release/fse.node
ld: warning: text-based stub file /System/Library/Frameworks//CoreFoundation.framework/CoreFoundation.tbd and library file /System/Library/Frameworks//CoreFoundation.framework/CoreFoundation are out of sync. Falling back to library file for linking.
ld: warning: text-based stub file /System/Library/Frameworks//CoreServices.framework/CoreServices.tbd and library file /System/Library/Frameworks//CoreServices.framework/CoreServices are out of sync. Falling back to library file for linking.
ld: warning: text-based stub file /System/Library/Frameworks//CFNetwork.framework/Versions/A/CFNetwork.tbd and library file /System/Library/Frameworks//CFNetwork.framework/Versions/A/CFNetwork are out of sync. Falling back to library file for linking.
ld: warning: text-based stub file /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/OSServices.framework/Versions/A/OSServices.tbd and library file /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/OSServices.framework/Versions/A/OSServices are out of sync. Falling back to library file for linking.
ld: warning: text-based stub file /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/LaunchServices.framework/Versions/A/LaunchServices.tbd and library file /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/LaunchServices.framework/Versions/A/LaunchServices are out of sync. Falling back to library file for linking.
ld: warning: text-based stub file /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/SharedFileList.framework/Versions/A/SharedFileList.tbd and library file /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/SharedFileList.framework/Versions/A/SharedFileList are out of sync. Falling back to library file for linking.
npm WARN notsup Unsupported engine for watchpack-chokidar2@2.0.0: wanted: {"node":"<8.10.0"} (current: {"node":"12.16.3","npm":"6.14.4"})
npm WARN notsup Not compatible with your version of node/npm: watchpack-chokidar2@2.0.0
npm WARN @adobe/aio-lib-ims-jwt@1.0.0 requires a peer of @adobe/aio-lib-ims@^1.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN inquirer-autocomplete-prompt@1.0.2 requires a peer of inquirer@^5.0.0 || ^6.0.0 but none is installed. You must install peer dependencies yourself.

+ @adobe/aio-cli@3.4.1
added 60 packages from 39 contributors, removed 13 packages, updated 15 packages and moved 18 packages in 34.702s

sarah:~ sarahxxu$ aio --version
@adobe/aio-cli/3.4.1 darwin-x64 node-v12.16.3
```

#### Steps to Reproduce

Listed above

#### Platform and Version

```
sarah:~ sarahxxu$ aio info

  System:
    OS: macOS High Sierra 10.13.6
    CPU: (4) x64 Intel(R) Core(TM) i5-7267U CPU @ 3.10GHz
    Memory: 18.34 MB / 8.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 12.16.3 - /usr/local/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.14.4 - /usr/local/bin/npm
  Virtualization:
    Docker: Not Found
  npmGlobalPackages:
    @adobe/aio-cli: 3.4.1
```

**Comment by shazron** [comment]
`aio-cli` was never supposed to be in the list of plugins to update and this has been fixed in #137 (released in 3.4.0)

We only have the warning that there is a new cli to update, and the user has to update the cli manually (you can't update the CLI when it is running itself -- it has to be out of band)

**Comment by sarahxxu** [comment]
Ah thanks a lot for the note! To jolt my memory, this warning is shown when there's a new cli version, and with any command the users run, this warning asks them to run the npm update command? Do you have an example? I'd like to update the docs accordingly!

**Comment by shazron** [comment]
this is printed before a command's output (for a max 7 days)
```
$ aio plugins --help
 ›   Warning: @adobe/aio-cli update available from 3.3.0 to 3.4.1.
 ›   Run npm install -g @adobe/aio-cli to update.
```

**Comment by sarahxxu** [comment]
Thanks! Updated the docs and closing issue.
